server/middleware/roleBasedAccessControl.ts

```diff
@@ -25,7 +28,8 @@ const checkUserHasPermissionTo = (action: ApplicationAction) => authorisationMid
  * Returns true if the specified [ApplicationAction] can be satisfied with any of the specified user roles.
  */
 const userHasPermissionTo = (action: ApplicationAction, userRoles: string[]): boolean => {
-  return userRoles.some(role => (rolesForAction()[action] || []).includes(role as ApplicationRole))
+  const requiredRoles = rolesForAction()[action]
```

Comment:
Minor bug fix here to return true if the specified action has no roles required for it